inference-engine/src/gna_plugin/frontend/scale_factor_calc.hpp

```diff
@@ -647,9 +647,14 @@ class ScaleFactorPerLayer<InferenceEngine::WeightableLayer*> {
             double weights_reducer = 1.0;
             auto conv = dynamic_cast<InferenceEngine::ConvolutionLayer *>(wl);
             if (conv) {
-                auto dims = conv->insData.front().lock()->getDims();
-
-                weights_reducer = MAX_VAL_2B_FEAT * scaleRange * dims[1] / std::numeric_limits<int32_t>::max();
+                auto inputs = conv->insData.front().lock();
+                auto dims = inputs->getDims();
+                double max_output_value = MAX_VAL_2B_FEAT * scaleRange * conv->_kernel_x * conv->_kernel_y;
```

Comment:
This change will reduce accuracy for already existing cases.

Reply:
I removed this change since it really causes accuracy reduction for some models and I'm not sure that the problem reproduced in the test will be actual for real models